exporting/pubsub/README.md

```diff
@@ -14,6 +14,9 @@ To use the Pub/Sub service for metric collecting and processing, you should firs
 Pub/Sub support is also dependent on the dependencies of those libraries, like `protobuf`, `protoc`, and `grpc`. Next,
 Netdata should be re-installed from the source. The installer will detect that the required libraries are now available.
 
+> `grpc` 1.32 or higher [cannot be used](https://github.com/netdata/netdata/issues/10193) to compile Netdata with Pub/Sub
```

Comment:
It is possible I am using some Latin pattern with English, but @joelhans should we use `Grpc` or an article `The` (Suggestion `The grpc must be version `) before the package name, to force an uppercase here?

Reply:
i think the library/pkg name is [`grpc`](https://github.com/grpc/grpc)

Reply:
Considering the package name, probably it is necessary an article. :thinking: , let us see what @joelhans will say.

Reply:
Software package names in English are proper names, so they never take articles (definite or otherwise) except in very rare cases for special emphasis of singular instances (just like you almost never use a/as/o/os in Potuguese with proper names).

Reply:
@Ferroin is correct about articles. But more importantly (to me), I would rather this sentence be active rather than passive:

```suggestion
> You [cannot compile Netdata](https://github.com/netdata/netdata/issues/10193)with Pub/Sub using `grpc` 1.32 or higher.
```

Reply:
Thak you @joelhans and @Ferroin ! :)